sdk/storagecache/Microsoft.Azure.Management.StorageCache/tests/OperationsTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Management.StorageCache.Tests
+{
+    using System.Collections.Generic;
+    using Microsoft.Azure.Management.StorageCache.Models;
+    using Microsoft.Azure.Management.StorageCache.Tests.Fixtures;
+    using Microsoft.Azure.Management.StorageCache.Tests.Utilities;
+    using Microsoft.Azure.Test.HttpRecorder;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    /// <summary>
+    /// Defines the <see cref="OperationsTests" />.
+    /// </summary>
+    [Collection("StorageCacheCollection")]
+    public class OperationsTests
+    {
+        /// <summary>
+        /// Defines the testOutputHelper.
+        /// </summary>
+        private readonly ITestOutputHelper testOutputHelper;
+
+        /// <summary>
+        /// Defines the Fixture.
+        /// </summary>
+        private readonly StorageCacheTestFixture fixture;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OperationsTests"/> class.
+        /// </summary>
+        /// <param name="testOutputHelper">The testOutputHelper<see cref="ITestOutputHelper"/>.</param>
+        /// <param name="fixture">The Fixture<see cref="StorageCacheTestFixture"/>.</param>
+        public OperationsTests(ITestOutputHelper testOutputHelper, StorageCacheTestFixture fixture)
+        {
+            this.fixture = fixture;
+            this.testOutputHelper = testOutputHelper;
+        }
+
+        /// <summary>
+        /// Verify the list of operations available to this subscription.
+        /// </summary>
+        [Fact]
+        public void TestOperationsList()
+        {
+            testOutputHelper.WriteLine($"Running in {HttpMockServer.GetCurrentMode()} mode.");
+            using (StorageCacheTestContext context = new StorageCacheTestContext(this))
+            {
+                var client = context.GetClient<StorageCacheManagementClient>();
+                client.ApiVersion = StorageCacheTestEnvironmentUtilities.APIVersion;
+
+                ApiOperationDisplay stopOperationDisplay = new ApiOperationDisplay("Stop the cache", "Microsoft Azure HPC Cache", "Cache", "Stops the cache");
+                ApiOperationListResult result = OperationsExtensions.List(client.Operations);
+
+                IList<MetricSpecification> metrics = new List<MetricSpecification>();
+                MetricSpecification metric = new MetricSpecification();
+                metric.AggregationType = "string";
+                metric.Unit = "unit";
+                metric.Name = "name";
+                metric.MetricClass = "metricClass";
+                metric.DisplayName = "displayName";
+                metric.DisplayDescription = "displayDescription";
+                MetricDimension dim = new MetricDimension();
```

Comment:
This can be simplified as well along with few others. dim/dims/aggrTypes

Reply:
Updated.